src/main/java/net/teamterminus/machineessentials/energy/electric/template/ElectricDeviceBlockEntity.java

```diff
@@ -21,49 +21,49 @@ public void tick() {
         super.tick();
         //reset counters
         ampsUsing = 0;
-        averageAmpLoad.increment(world,0);
-        averageEnergyTransfer.increment(world,0);
+        averageAmpLoad.increment(world, 0);
+        averageEnergyTransfer.increment(world, 0);
 
         //try to pull max allowed current from any connected side
         for (Direction dir : Direction.values()) {
             BlockEntity be = MachineEssentials.getBlockEntity(dir, world, this);
-            if(be instanceof ElectricWire){
-                receiveEnergy(dir,getMaxInputAmperage());
+            if (be instanceof ElectricWire){
+                receiveEnergy(dir, getMaxInputAmperage());
             }
         }
     }
 
     @Override
     public long receiveEnergy(@NotNull Direction dir, long amperage) {
-        if(amperage > getMaxInputAmperage()){
+        if (amperage > getMaxInputAmperage()){
```

Comment:
If we are talking about putting spaces everywhere then we also should put space here after `if ()`